source/lib/assertions/assert.ts

```diff
@@ -72,6 +72,15 @@ export const expectNotDeprecated = (expression: any) => {
 	// Do nothing, the TypeScript compiler handles this for us
 };
 
+/**
+ * Assert that the `expression` provided is `never`.
```

Comment:
The doc comment is inconsistent with the readme. The readme talks about a "value" and this talks about "expression". Make sure both are in sync.

Reply:
I just copied this part. I think your comment applies to the all other functions as well

Reply:
It does indeed, but doesn't mean your PR should not do the right thing.

Reply:
https://github.com/SamVerschueren/tsd/pull/157